Makefile

```diff
@@ -1,8 +1,13 @@
-.PHONY: default all OpenShift OCS CNV
```

Comment:
The script in the CNV dir has changed, so this file needs an update for that. Because of the script names changing, I was thinking a Makefile per directory could make sense, then the root level Makefile just pushd and calls make. So if you raise a PR to change a script name, you need only mess with the Makefile in your directory instead of worrying what other Makefiles call your scripts